cmd/core/api.go

```diff
@@ -36,6 +38,7 @@ func CreateApiServer(bpfObjects *ebpf.BpfObjects, pfcpSrv *PfcpConnection, forwa
 		v1.GET("config", DisplayConfig())
 		v1.GET("xdp_stats", DisplayXdpStatistics(forwardPlaneStats))
 		v1.GET("packet_stats", DisplayPacketStats(forwardPlaneStats))
+		v1.POST("edit_config", EditConfig)
```

Comment:
If possible use `config` name for this request (but POST method). So we have `config` API for get config values. And the same method name for set config values.

Reply:
Done